frontend/components/base/Table.vue

```diff
@@ -9,18 +9,19 @@
         <div class="table-inner-container">
             <table :style="{ minWidth: numColumns * 125 + 'px' }">
                 <tr>
-                    <th v-if="editLink"></th>
+                    <th v-if="showEditLink"></th>
```

Comment:
much better naming 🙏

Reply:
The old naming made sense because `editLink` literally *was* the static link to the edit page. Now that the navigation is done in an event handler, it makes more sense just to move it to a boolean.